examples/dataflow-python-examples/CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+Thanks for your interest in contributing to the dataflow Python examples!
+
+Here is the general guidance:
+
+* Batch Pipelines should live in batch-examples folder
+
+* Streaming Pipelines should live in Streaming-examples folder
+
+* Pipelines written using single python module without any other associated scripts should be added to either 
+   batch-examples/cookbook-examples or streaming-examples/cookbook-examples depending on the nature of pipeline.
+  
+*  Pipelines written either over multiple modules or has associated scripts (e.g: shell scripts/ terraform etc) must be
+   in its own directory as a sub directory under either batch-examples/ or streaming-examples/ depending on the nature of pipeline.
+   
+   Every self contained pipeline should follow below guidelines:
+    
+    1. Include requirements.txt specifying dependent python packages with respective versions 
+    
+    2. Include cloudbuild.yaml file containing 2 steps:
```

Comment:
I worry a bit about having two conflicting "contributing" steps, but makes sense to have some language / platform specific guides.

There's one potential issue with the way we are doing Cloud Build, which is that it seems like if there is any failure it'll cancel the whole pipeline, but I think since we actually trigger a Cloud Build job per cloudbuild.yaml it should be able to continue running the job even if we don't see the logs in the root job after one failure.

Reply:
Initially I was leaning more towards having a single cloudbuild.yaml file for all examples but issue with that approach is all pipelines must be enforced to use same package versions

Reply:
Not having the failure reason in the logs is far from an ideal contributing flow and will leave contributors asking reviewers to help them figure out why their check is failing.
In a future PR, could we write some helper that aggregates the logs from all children cloud build jobs run? this might be easier said than done.

Reply:
Sure. We can explore that option..